pkg/csi/service/wcpguest/controller_helper.go

```diff
@@ -202,6 +205,48 @@ func getPersistentVolumeClaimSpecWithStorageClass(pvcName string, namespace stri
 	return claim
 }
 
+// generateGuestClusterRequestedTopologyJSON translates the topology into a json string to be set on the supervisor
```

Comment:
can you add some unit test cases for the following two util func? I believe Divyen added that in the change in customized external-provisioner. You can just copy it over.